skbeam/core/mask.py

```diff
@@ -108,3 +109,81 @@ def threshold_mask(images, threshold, mask=None):
         if len(bad_pixels[0]) != 0:
             mask[bad_pixels] = 0
         yield mask
+
+
+def mask_edge(img_shape, edge_size):
+    """
+    Mask the edge of an image
+
+    Parameters
+    -----------
+    img_shape: tuple
+        The shape of the image
+    edge_size: int
+        Number of pixels to mask from the edge
+    Returns
+    --------
+    1darray:
+        The raveled mask array, bad pixels are 0
+    """
+    mask = np.ones(img_shape)
+    mask[:, :edge_size] = 0
+    mask[:, -edge_size:] = 0
+    mask[:edge_size, :] = 0
+    mask[-edge_size:, :] = 0
+    return mask.ravel().astype(bool)
```

Comment:
I preferred the 1D masks because it allows you to make a much smaller image to integrate
```python
fimg = img.ravel()
maksed_img = fimg[mask]
integrated = scipy.stats.binned_statistic
```